brainflow-dev/brainflow
# add unicorn for windows

Signed-off-by: Andrey Parfenov <a1994ndrey@gmail.com>

#134 

Steps to test:
* install visual studio 2017
* in VS installer make sure you selected ATL support(need for altstrings), tab called 'individual components'
* Run commands below

```
git clone https://github.com/brainflow-dev/brainflow
cd brainflow
git checkout unicorn_windows
tools\build_win64.cmd
cd python-package
python -m pip install -U .
python ..\tests\python\brainflow_get_data.py --log --board-id 8
```
